wordpress-mobile/WordPress-iOS
# Improve VoiceOver integration in Menus section - Mark I

Fixes parts of #12881

This PR improves VoiceOver integration of the Header part of the Menus section.

![Menus](https://user-images.githubusercontent.com/9772967/90129707-6db36780-dd69-11ea-8d49-467386338161.png)

VoiceOver reads the Areas picker cell like this:
> Menu area: Header. 3 menu areas in this theme. Button. [hint] Expands to select a different menu area.

I'm not sure about the `3 menu areas in this theme` part. It might be useful information but makes the label a bit too long.
What do you think @iamthomasbishop ?

Same with the Menu picker cell. It reads:
> Menu in area Header: Primary. 3 menus available. Button. [hint] Expands to select a different menu to edit.


### To test:

[Using Voice Over](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/docs/using-voiceover.md):
- Go to the Menu section of a site.
- Check that the header section of the controller reacts as expected to VoiceOver.
- Expand the Areas and Menus section.
- Also test switch actions (swipes to the left and right).
- Test landscape where Areas and Menus are side by side.
  - Check that the areas and menus are navigated as a group via switch.
- Check out the [guidelines](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/docs/voiceover-guidelines.md#questions-to-ask).

cc @designsimply 

PR submission checklist:
